code/modules/client/verbs/who.dm

```diff
@@ -100,13 +103,30 @@
 			if(C.is_afk())
 				msg += " (AFK)"
 			msg += "\n"
+
+	// for standard players
 	else
+		var/list/admin_list = list()
+		var/list/non_admin_list = list()
 		for(var/client/C in GLOB.admins)
 			if(C.is_afk())
 				continue //Don't show afk admins to adminwho
 			if(!C.holder.fakekey)
-				var/rank = "\improper [C.holder.rank]"
-				msg += "\t[C] is \a [rank]\n"
+				if(check_rights_for(C, R_ADMIN)) // ahelp needs R_ADMIN. If they have R_ADMIN, they'll be listed in admin list.
+					var/rank = "\improper [C.holder.rank]"
+					admin_list += "\t[C] is \a [rank]\n"
+				else // admins without R_ADMIN perm should be sorted in different area, so that people won't believe coders will handle ahelp
+					var/rank = "\improper [C.holder.rank]"
+					non_admin_list += "\t[C] is \a [rank]\n"
+
+		msg = "<b>Current Admins:</b>\n"
+		for(var/each in admin_list)
+			msg += each
+		if(length(non_admin_list)) // notifying the absence of non-admins has no point
+			msg += "<b>Current Non-admin staffs:</b>\n"
```

Comment:
```suggestion
			msg += "<b>Other Staff:</b>\n"
```

Reply:
Currently there are only three branches of staff: 
* Admin Team
* Coding Team
* Mentor Team

There's no reason to label this as "non-admin staff" separate from mentors. It contains exclusively maintainers and one head coder.